app/lib/api_particulier/api.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
```

Comment:
Oui, ça vaudrait le coup de le généraliser. Le langage & Rubocop nous y incitent et ce comportement devait être la norme en Ruby 3, mais l'idée a finalement été abandonnée ; non pas parce qu'elle est mauvaise, mais pour le risque d'incompatibilité que cela implique d'activer ce _magic comment_ par défaut.

> I consider this for years. I REALLY like the idea but I am sure introducing this could cause HUGE compatibility issue, even bigger than Ruby 1.9. So I officially abandon making frozen-string-literals default (for Ruby3).
>
> This does not mean we are going to remove the frozen-string-literal feature that can be specified by magic comments.
>
> Matz.

source: https://bugs.ruby-lang.org/issues/11473#note-53

Note: l'ajout de cette ligne sur l'ensemble des fichiers peut se faire à l'aide de l'option `autocorrect` de Rubocop :

```
rubocop --only Style/FrozenStringLiteralComment -a
```

source: https://docs.rubocop.org/rubocop/1.0/cops_style.html#stylefrozenstringliteralcomment